Docs/Configuration.tex

```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
 
 This document provides information on
 \href{https://github.com/acidanthera/OpenCorePkg}{OpenCore} user
-configuration file format used to setup the correct functioning of macOS
+configuration file format used to setup the correct functioning of the macOS
 operating system. It is to be read as the official clarification of expected
 OpenCore behaviour. All deviations, if found in published OpenCore releases,
-shall be considered documentation or implementation bugs, and are requested to be
-reported through \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
-Errata sheet is available in
+shall be considered to be documentation or implementation bugs which should be
+reported via the \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
+An errata sheet is available in
```

Comment:
Errata is the plural form or erratum, so *Errata sheet* should be fine.

Reply:
I think an article refers to the noun (sheet) here with errata being treated as an adjective. So either "an errata sheet" or "errata sheets", but I believe the former is more common. Omitting an article in such cases is probably allowed for the manual speech, but better to have it. So I believe the change is correct.